protocol/protocol.go

```diff
@@ -201,7 +204,6 @@ func Decode(bytes []byte) (map[uint32][]byte, error) {
 		payloads = payloads[length:]
 		ret[tag] = payload
 		currentOffset = nextOffset
-		lastTag = tag
```

Comment:
Are sorted tags a requirement or not?
Related: https://github.com/aanchal4/draft-roughtime/pull/25

Reply:
We accidentally omitted it from the draft.